src/errors/Messages.js

```diff
@@ -58,6 +58,14 @@ const Messages = {
   SELECT_OPTION_VALUE: 'MessageSelectOption value must be a string',
   SELECT_OPTION_DESCRIPTION: 'MessageSelectOption description must be a string',
 
+  TEXT_INPUT_CUSTOM_ID: 'TextInputComponent customId must be a string',
+  TEXT_INPUT_LABEL: 'TextInputComponent label must be a string',
+  TEXT_INPUT_PLACEHOLDER: 'TextInputComponent placeholder must be a string',
+  TEXT_INPUT_VALUE: 'TextInputComponent value must be a string',
+
+  MODAL_CUSTOM_ID: 'Modal customId must be a string',
+  MODAL_TITLE: 'Modal title must be a string',
```

Comment:
Can't all of these just be INVALID_TYPE?

Reply:
Maybe, I was just maintaining consistency with the fact that buttons and selects all have unique errors.

But I wrote those too so it could have been just as bad an idea there.

Reply:
We might need a separate PR later on to clear some of these errors up in v14 but IMO these can be left out in favor of INVALID_TYPE for now